### PR TITLE
Export data format

### PR DESCRIPTION
**Pull request by shaobinspire**

### CODE REVIEW

File: Applications/Spire/Source/Common/ExportTable.cpp [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
 
+using namespace Beam::TimeService;
 using namespace boost::gregorian;
 using namespace boost::posix_time;
 using namespace Nexus;
 
+namespace {
+  auto to_string(ptime time) {
+    auto local_time = ToLocalTime(time);
+    auto ss = std::ostringstream();
```
Comment (kranar) [inline Applications/Spire/Source/Common/ExportTable.cpp:13]: Avoid constructing this over and over for every call to ```to_string```.

File: Applications/Spire/Source/SpireTester/ExportTableTester.cpp [hunk 2]
```diff
@@ -1,40 +1,68 @@
 #include <doctest/doctest.h>
+#include "boost/date_time/c_local_time_adjustor.hpp"
```
Comment (kranar) [inline Applications/Spire/Source/SpireTester/ExportTableTester.cpp:2]: Not a proper include.

File: Applications/Spire/Source/SpireTester/ExportTableTester.cpp [hunk 3]
```diff
@@ -1,40 +1,68 @@
...
 #include "Spire/Spire/ExportTable.hpp"
 #include "Spire/Ui/CustomQtVariants.hpp"
 
+using namespace boost;
+using namespace boost::gregorian;
 using namespace boost::posix_time;
 using namespace Nexus;
 using namespace Spire;
 
+auto to_string(ptime t) {
```
Comment (kranar) [inline Applications/Spire/Source/SpireTester/ExportTableTester.cpp:16]: What is the purpose of this function? Look at removing it.

Comment (shaobinspire) [reply Applications/Spire/Source/SpireTester/ExportTableTester.cpp:16]: The purpose of this function is to display the date of the time. The function of `displayText(ptime time)` in `CustomQtVariants` doesn't show the date of the current day.
```
QString Spire::displayText(ptime time) {
  auto local_time = ToLocalTime(time);
  auto current_time = ToLocalTime(posix_time::second_clock::universal_time());
  if(local_time.date() == current_time.date()) {
    return QString::fromStdString(to_simple_string(local_time).substr(12));
  }
  return QString::fromStdString(to_simple_string(local_time));
}
```

Comment (shaobinspire) [reply Applications/Spire/Source/SpireTester/ExportTableTester.cpp:16]: I will think about how to remove it.

**Pull request merged**